enableHelpTips = true;
helpTips = {
    analyticalreplicateview: {
        "treatment_id": "<b>[required]</b> Describe any protocol followed or treatment applied over the biological replicate in order to isolate a set one or more of Analytical Sample, potentially used later during the sequencing step.</br>E.g. Sampling in Metabolomics, RNA Library extraction in RNA-Seq, Prefractionation treatment in Proteomics,...",
        "analytical_rep_id": "Identify uniquely each Analytical Sample for the current Biological Replicate and Biological Condition [<i>Autogenerated</i>].",
        "analytical_rep_name": "<b>[required]</b> An <i>human readable</i> name for each Analytical Sample."
    },
    batchcreationwindow: {
        "batch_name": "<b>[required]</b> An <i>human readable</i> name for each Experimental batch.</br>Suggestion: , e.g. <i>.</i>",
        "batch_creation_date": "<b>[required]</b>Date in which the cultivation process was carried out.",
        "description": "A complete description for cultivation process in which a set of Biological Replicates were obtained."
    },
    bioreplicateview: {
        "batch_name": "Identify the cultivation process in which this Biological Replicate was obtained.",
        "treatment_id": "<b>[required]</b> Describe any protocol followed or treatment applied over the biological replicate in order to isolate a set one or more of Analytical Sample, potentially used later during the sequencing step.</br>E.g. Sampling in Metabolomics, RNA Library extraction in RNA-Seq, Prefractionation treatment in Proteomics,...",
        "analytical_rep_id": "Identify uniquely each Analytical Sample for the current Biological Replicate and Biological Condition [<i>Autogenerated</i>].",
        "analytical_rep_name": "<b>[required]</b> An <i>human readable</i> name for each Analytical Sample."
    },
    experimentdetailsview: {
    },
    rawdataview: {
        /*GENERAL RAW DATA FIELDS*/
        "technical_rep_set_id": "The identifier of the technical replicate set which this Raw data belongs.",
    },
    extractionmethodview: {
        /*SEQUENCING AND OTHER FIELDS*/
        "extracted_molecule": "Specify the type of molecule that was extracted from the biological material. Eg. total RNA, cytoplasmic RNA, nuclear RNA,..",
        /*DNAseseq*/
        "separation_method_type": "Specify the restriction enzyme used.",
        /*COLUMN CHROMATOGRAPHY*/
        "cc_sample_description": "A description of the source, such as means of collection, volume, concentration or previous step of processing. Include information about tags, molecular mass, pH and anything else relevant to experiment. In a multi-dimensional separation, this should contain the name of the fraction from the previous dimension.",
        "cc_sample_processing": "A brief description of processing applied to the sample, including the buffer, the volume, any pre-treatments (such as the addition of an acid to facilitate binding) and any tag applied e.g. for fluorescence detection.",
        "cc_sample_injection": "A description of sample injection procedure, such as details of the volume, inline/offline, direct/loop (full/partial loop), flush conditions, and sample storage temperature.",
        "column_manufacturer": "The name of the column manufacturer.",
        "column_model": "The model of the column.",
        "separation_mode": "A description of the type of column being used. (e.g., Separation mechanism: Affinity, Anion Exchange, Cation Exchange, Reverse Phase, Hydrodynamic Volume).",
        "dimensions": "The dimensions of the column in terms of length and inner diameter or the total volume of the column where this value is appropriate, for example to describe gel filtration experiments.",
        "description_of_stationary_phase": "A description of the constituents of the stationary phase, including the name of the packing material and the particle size, or describe packing materials in the case of monoliths including their manufacturer if stationary phase obtained separately from column.",
        "additional_accessories": "Details of any guards or traps used in conjunction with the column.",
        "combined_unit_manufacturer": "The name of the manufacturer of combined unit or component parts including software used to operate.",
        "combined_unit_model": "The model name provided by the manufacturer.",
        "mobile_phases": "Describe all the mobile phases of the run.",
        "mp_name": "<b>[required]</b> Name used to refer the mobile phase in the <i>Column Run properties</i> section.",
        "mp_description": "For each constituent a description, the concentration, pH (and how adjusted) and date of preparation.",
        "time": "The total time of the Column Run with appropriate units.",
        "gradient": "The proportion of each of the mobile phases, relative to time , for the function describing the gradient, including its overall duration. There may be several steps that together make up the gradient",
        "flow_rate": "The flow rate at which the mobile phase is applied to the column, including the time period for which this holds if it varies during the experiment.",
        "temperature": "The temperature at which the column is run, including the time period for which this holds if it varies during the experiment.",
        "pre_run_process_type": "A description of the purpose of the process, such as equilibration, calibration or washing (this may be part of the column run, as one step or as a preconditioning of the column prior to use).",
        "pre_run_process_substance": "A description of the reagent used in the process.",
        "pre_run_process_time": "The duration of the process.",
        "pre_run_process_flowrate": "The rate at which the mobile phase is applied to the column.",
        "detection_equipment": "Manufacturer and model, or description.",
        "detection_type": "A description of the kind of detector (e.g. UV, MS).",
        "detection_equipment_settings": "A description of control properties of the detector, such as the wavelength that is being detected.",
        "detection_timescale": "The time range covered by the trace produced by the detector.",
        "detection_trace": "The location and format of the trace if appropriate.",
        "fractions": "Describe all the fractions of the run (if separation purpose is preparative).",
        "fr_name": "<b>[required]</b>Name, unique within a run, by which a fraction can be referenced.",
        "fr_description": "Either a description of the procedure by which the fractions were collected i.e. start/end time, size (time or volume), mode (fixed or peak directed), or a description of the individual fractions (e.g. time of collection, volume).",
        /*MASS SPECTROMETRY DETAILS*/
        "mass_spectrometer_manufacturer": "The manufacturing company and model name for the mass spectrometer.",
        "customizations": "Any significant (i.e. affecting behaviour) deviations from the manufacturer’s specification for the mass spectrometer.",
        "ionization_source": "Choose the Ion source",
        "supply_type": "Whether the sprayer is fed by, for example, chromatography or CE or is loaded with sample once before spraying, such as in static infusion",
        "interface_manufacturer_and_model": "The manufacturing company and model name for the interface; list any modifications made to the standard specification. If the interface is entirely custom-built, describe it or provide a reference if available.",
        "sprayer_type_manufacturer_and_model": "The manufacturing company and model name for the sprayer; list any modifications made to the standard specification. If the sprayer is entirely custom-built, describe it briefly or provide a reference if available.",
        "other_electrospray_ionisation": "Where appropriate, and if considered as discriminating elements of the source parameters, describe these values.",
        "plate_composition": "The material of which the target plate is made (usually stainless steel, or coated glass); if the plate has a special construction then that should be briefly described and catalogue and lot numbers given where available.",
        "matrix_composition": "The material in which the sample is embedded on the target (e.g. alpha-cyano-4-hydroxycinnamic acid).",
        "psd_summary": "Confirm whether post-source decay, laser-induced decomposition, or in-source dissociation was performed; if so provide a brief description of the process (for example, summarize the stepwise reduction of reflector voltage).",
        "laser_type_and_wavelength": "The type of laser and the wavelength (nm) of the generated pulse. For instance nitrogen, 337 nm.",
        "other_maldi": "Other details of the laser used to shoot at the matrix-embedded sample if considered important for the interpretation of data; this might include the pulse energy, focus diameter, attenuation details, pulse duration at full-width half maximum, frequency of shots in Hertz and average number of shots fired to generate each combined mass spectrum.",
        "other_ionization_description": "Describe the ion source and provide relevant and discriminating parameters for its use.",
        "mass_analyzer_type": "Describe the analyzer(s) used in the MS experiment. Example might be MS1 survey scans in an Orbitrap and MSn analysed in a linear trap",
        "reflectron_status": "Time-of-Flight drift tube: Reflectron status (on, off, none) when status can be set.",
        "activation_location": "The hardware element where activation and/or dissociation occurs. For instance a quadrupole collision cell, a 3D ion trap, the ion source (for ISD, PSD, LID, isCID)",
        "gas_type": "The composition and pressure of the gas used to fragment ions, for instance in the collision cell.",
        "activation_type": "The type of activation and/or dissociation used in the fragmentation process. Examples might include Collision Induced Dissociation (CID) with a static or spread collision energy, Electron Transfer Dissociaton (ETD) with provided activator molecules.",
        "acquisition_software": "The instrument management and data analysis package name and version; where there are several pieces of software involved, give name, version and role for each of them. Mention also upgrades not reflected in the version number.",
        "acquisition_parameters": "The information on how the MS data have been generated. It describes the instrument’s parameter settings / acquisition method file or information describing the acquisition conditions and settings of the MS run. Ideally this should be a URI+filename, for example an export of the acquisition method (and Tune page or other relevant information where appropriate). This includes for instance if the data are acquired and primarily stored in a profile or centroid mode. An explicit text description of the acquisition process is also desirable. This includes the acquisition sequence (for instance as simplified as top-five method with a cycle made of one full MS1 scan in the Orbitrap, followed by a precursor selection of 5 most intense ions applying an exclusion window of 30 seconds and followed by the acquisition of 5 product ion scans generated in the LTQ analyzer, and detected in the LTQ). This allows to differentiate between the use of a selected precursor window vs unselected fragmentation (MS`E/bbCID/AIF). This also allows to explicitly describe pre-defined the acquisition method of a SRM experiment where all transitions and detection windows are specified.",
        "analysis_software": "The MS data analysis package name and version; where there are several pieces of software involved, give name, version and role for each one. Mention also upgrades not reflected in the version number.",
        "analysis_parameters": "The information on how the spectra have been processed. This include the list of parameters triggering the generation of peak lists, chromatograms, images from raw data or already processed data and the order in which they have been used. This can be a list or a parameters file.",
        "source_location": "The location and filename under which the original raw data file(s) from the mass spectrometer and the processed file(s) are stored. Also give the type of the file where appropriate. Ideally this should be a URI+filename.",
        "intensity_values": "The actual data (m/z and intensity) for each spectrum. This is most often provided in the spectra and/or peaklist file.",
        "ms_level": "The MS level (e.g. MS^2) at which each spectrum was acquired. This is most often provided in the spectra or peaklist file.",
        "ion_mode": "The ion mode (positive or negative). This is most often provided in the spectra or peaklist file.",
        "additional_info": "For tandem spectra; in addition to the preceding information,the precursor m/z value and the charge state of the precursor ion should be given; the mass spectrum used to deduce the precursor information should also be provided.",
        /*Nuclear magnetic resonance*/
        "instrument_manufacturer": "The manufacturing company for the instrument.",
        "instrument_model": "The model for the instrument.",
        "strength": "Magnetic field strength in Tesla or proton resonance frequency.",
        "console_description": "",
        "vt_control": "",
        "pulsed_field_strength": "Describe the pulsed field gradients (z or x,y,z)",
        "max_gradient_strength": "Describe the maximum gradient strength (if used)",
        "n_shims": "Number of shims.",
        "n_channels": "Number of Channels.",
        "probe_type": "Describe the probe type. E.g. 10mm 31^P, 5 mm HCN cold probe, 3 mm flow-probe etc.",
        "sample_state": "Solution or solid-state.",
        "operation_mode": "Automation or manual operation.",
        "tune_mode": "Auto-tune or manual tune.",
        "probe_gas": "",
        "volume": "Volume of prepared sample.",
        "final_sample_status": "State of the sample after preparation.",
        "nmr_tube_type": "Type of NMR tube (e.g. conventional, Shigemi, mircrocell etc.)",
        "pH": "Value of pH for the sample.",
        "solvent": "Solvent (if any). E.g. D20, CD3OD, CDCl3 etc.)",
        "buffer": "",
        "resonant_frequency": "",
        "acquisition_description": "Description for the data acquisition step (acquisition parameters).",
        /*CAPILLARY ELECTROPHORESIS*/
        "experiment_type": "The CE mode e.g. CZE, MEKC, CGE CIEF, CEC, ITP and mixed mode (e.g. transient ITP-CZE), preconcentration, stacking.",
        "experiment_aim": "Quantitation, identification, size determination, pI determination, enantiomeric excess determination, etc.",
        "ce_sample_description": "Concentration of sample(s) (if known) including any label, marker or tag applied that will be used for detection, such as fluorescent labels (by name only). Identify and give source to possible controls, system suitability standard, sample related standards, tEOF-marker and test samples. If calibrants, state concentrations of materials",
        "ce_sample_solution": "The components, with concentrations and pH (excluding the sample itself) of the sample solution that is to be injected into the capillary (e.g. background electrolyte). Manufacturer, order and lot numbers used.",
        "ce_sample_preparation": "Has the sample undergone any further processes e.g. centrifugation, filtration, ultrasonication etc.",
        "capillary_description": "Type of capillary being used; e.g. fused silica capillary, coated capillary, packed capillary, gel-filled capillary etc.",
        "capillary_source": "If the capillary was purchased pre-made (e.g. coated, with window, or to pre-cut lengths) then include the model name, catalogue number, manufacturer and lot number. If the capillary has been manufactured in house then supplier of silica capillary, catalogue number and lot number should be given. </br>If using a coating, gel filled, packed or monolithic capillary, protocol, the manufacturer, catalogue number and lot number, type, surface modifications, particle and pore size, gels should be given as appropriate. In addition reference should be given to the method and if no published protocol is available a process should be outlined.  </br>If polyimide coating has been removed from the ends of the capillary this should be described.</br>",
        "capillary_dimensions": "The exact dimensions of the capillary employed: From inlet to detection window (effective length, cm); from inlet to outlet (total length, cm); and the inner and outer diameters of the capillary (µm).",
        "ce_temperature": "Controlled temperature of capillary (if controllable).",
        "auxiliary_data_channels": "Descriptions of the auxiliary channels set up to monitor current, power, voltage and pressure applied and values obtained for all steps and state  if this is to be used as  indication of system suitability.",
        "duration": "Duration of data collection from detector and auxiliary data channels.",
        "run_description": "Protocol for the run. Normally follows steps such as (i) preconditioning prior to the first use of a capillary (various flush steps, designed to clean / activate / coat the inner walls of the capillary), (ii) preconditioning (carried on each analysis); (iii) injection; (iv) separation; (v) post-conditioning (again, various flush steps). Each of these steps should be defined as follows:<ul><li>Step name: Descriptor for an individual step in the run. This includes: pre-conditioning, flush, wash, injection, focusing, mobilization and separation steps.</li><li>Step conditions: The conditions applied to the capillary; e.g. pressure or voltages all relative to time and vial locations and contents. This information should include voltage mode (positive/negative, step and hold, or gradient) if applicable. The frequency of vial and/or mobile phase/buffer/gel exchange replenishment intervals should also be detailed. For CIEF mobilization post focusing, state what mode of peak migration used e.g. pressure or vacuum or ionic</li><li>Electrolyte/ampholytes solutions description: for Pre-conditioning, flush, wash and background electrolyte / ampholytes solutions in terms of components with concentrations. pH adjustments made should also be described including temperature. Specify also if solutions have been filtered to remove particulate matter (commonly 0.20-0.45 µm filters used) or ultrasonicated to degas.</li><li>Sample injection description:  Volume of sample solution placed in injection vial (µL), controlled temperature of sample storage (if possible), injection geometry (state whether short or long end injection)…</li></ul>",
        "step_owners": "<b>[required]</b> Only owners can edit the information of this step."
    },
    stepview: {
        "motivation": "Describes the objectives of the current step.",
    },
    intermediate_data_specificdetailspanel: {
        //SPECIFIC DETAILS FOR INTERMEDIATE DATA STEP
        "feature_type": "Describe the type of extracted feature (e.g. Metabolites, peptides...).",
    },
    processed_data_specificdetailspanel: {//SPECIFIC DETAILS FOR PROCESSED DATA
        /*MASS SPECTROMETRY QUANTIFICATION*/
        'groups': "Description of the experimental sample grouping including group names (if appropriate). A group is a sample set that corresponds to a certain experimental condition with its value (i.e. condition “gender” with value “female” is group “gender_female”).",
        'replicates': "Description of the biological and/or technical replicate structure (if appropriate). This is also a group structure like above, but hierarchically ordered. In the case of technical replicates, describe the kind of technical replicates: e.g., cell lysis  replicates: samples are replicated prior to cell lysis step, digestion replicates: samples are replicated prior to digestion step, LC-MS/MS replicates: samples are replicated prior to LC-MS/MS runs.",
        'labelling_protocol': "In the case of label-based quantification methods, describe the labelling protocol, including the labelling level: if labelling occurs at the element, amino acid, peptide (terminus) or protein (terminus) level.",
        'sample_description': "Description for the sample",
        'sample_name': "Unique name for the sample to be referred to elsewhere in the document.",
        'sample_amount': "Sample amount (including the appropriate units)",
        'sample_labelling': "In the case of label-based quantification methods, indicate which label is associated with which sample. If appropriate, provide the mass of the reporter ion (e.g. iTRAQ 4-plex : 114), the reagent (e.g., ICPL-12^C6) or the isotope labeled amino acid(s) used in this sample (e.g., 13^C6-Lysine and 13^C6-Arginine) or the isotope-labeled element (e.g. 15^N).",
        'replicates_and_groups': "Replicates and/or groups from sections 2.1.1 and 2.1.2 that the sample belongs to. One sample / assay is usually part of several replicates / groups, e.g. part of technical replicate X AND biological replicate Y, or part of “gender_male” AND “age_50-60”.",
        'isotopic_correction_coefficients': "In case of iTRAQ/TMT like labelling, provide the isotopic correction coefficients. In the case of other isotopic labels, percent enrichment, such as 98% 15^N.",
        'internal_references': "List of internal references used and their amount. Also state their specific purpose such as quantification, normalization or alignment.",
        'input_data_type': "Description of the type of the source data used for the quantitative analysis: Zoom scan, full MS scan, tandem MS, SRM chromatogram, etc…",
        'input_data_format': "Specify the input data format for the quantitation analysis, either post-processing formats such as mzIdentML, protXML, pepXML, dat, or if needed provide the original format such as Thermo raw, AB Sciex wiff, mzData, mzML, mzXML, mgf. For XML format data, provide the schema version number.",
        'input_data_merging': "In case of fractionation has been applied to the samples prior to the acquisition of MS data, specify which type of fractionation has been performed (e.g. chromatography-based, gel-based, etc.) and how many fractions/bands have been generated for each sample. State whether input data coming from each fractionation unit is merged in a single dataset prior to quantification analysis or not. In that case, describe the software or algorithm used for this purpose (name, manufacturer and version) or refer to the one described below.",
        'quantification_software': "For each software (or library) used in the quantitative analysis, provide the trade name together with the version name or number according to the original code and functionality of the software. In case of in-house developed tools provide a brief description of it together with a reference if available. Brief description of the selected analysis pipeline if it is customizable from the software.",
        'selection_method': "Description of the methods together with its relevant parameters used in the selection and/or matching of features: e.g., feature detection (m/z and retention time windows, applied filters by charge state or by signal to noise, etc…), pairs finding, chromatogram alignment, the iTRAQ reporter integration method (most confident centroid, most intense centroid, centroid sum over 20ppm window, etc…), or any other method performed to select and/or match features prior to their quantification. Also describe how the primary extracted quantification value for each feature and/or peptide is calculated (e.g., volume measurement, max peak intensity, etc…).",
        'confidence_filter': "Describe any filter threshold applied to the features (e.g. a threshold filter over the “seed score” in OpenMS software) or to the identification matches prior to quantification of features, such as FDR, FWER, score, probability thresholds, etc… State if applicable in which level has been applied (PSM, peptide or protein level). This includes the description of how the threshold has been determined (e.g. arbitrary, decoy strategy, binomial distribution, etc…).Also state whether quantitation restricted to unique (i.e. non-shared) peptides or whether some features/peptides have been excluded from the quantification (e.g. containing a non-quantitative modification like oxidized methionine, etc…).",
        'missing_values': "Describe the missing values imputation and outlier removal methods. Describe (if appropriate) what does a ‘0’ value means, that is, whether it means absent or just below the level of detection, etc…",
        'quantification_values_calculation': "Describe how quantification values and / or ratios have been calculated from values described in section 4.2, e.g., aggregation of all features per peptide (average, geometric average, summing, ANOVA analysis), description of the enumerator and denominator of the ratio, etc…",
        'replicate_aggregation': "Describe the calculation method aggregating / combining the values from peptide or experimental replicates and/or groups (e.g., average, geometric average, weighting, etc. ...), as well as any data transformation (e.g., logarithmization, multiplication, division, etc…).",
        'normalization': "Description of any normalization procedure, e.g. 'median peptide ratio for all measurements forced to 1' or 'each sample spiked with equal amount of peptide X or protein Y, as per 2.2.4'.",
        'protein_quantification_values_calculation': "Describe how protein quantification values and / or ratios have been inferred from peptide quantification values (especially in case of shared peptides). Specify additional criteria like: “take into account only proteins with at least two identified peptides” used in order to filter PSMs before rolling up them into protein abundance.",
        'specific_corrections': "Describe any protocol specific corrections like: “Arg-Pro conversion”.",
        'correctness_estimation_methods': "Describe how are calculated the values that describe the error potential of given / calculated quantification values (e.g., standard deviation, standard error, confidence interval, super-median of the experiment, standard error for the fit, standard error for the calculated ratio, coefficient of variation, pvalue, FDR, FWER, etc…). Also describe any applied threshold or filter based in these statistical values (e.g., p-values <= 0.05). Also describe if any threshold value has been used to consider that a feature/peptide/protein is over-expressed, and why that value has been selected.",
        'curves_calibration': "If internal references are used (described in section 2.2.4), provide the resulting calibration curves.",
        'primary_extracted_quantification_values': "The calculated primary extracted quantification values for each feature and also values representing their confidence (see above). The values can be provided either by a URI or the location of the resulting data files.",
        'primary_extracted_quantification_files_location': "The transformed / aggregated / combined quantification values for each peptide, calculated from the quantification values from the previous section (5.1.1) (e.g. medians, averages, standard deviations, super-medians, fold changes, p-values etc…). Also provide the values representing their statistical estimation of confidence (see above). The values can be provided either by a URI or the location of the resulting data files, or their availability.",
        'peptide_quantification_values': "The calculated primary extracted quantification values for each feature and also values representing their confidence (see above). The values can be provided either by a URI or the location of the resulting data files.",
        'peptide_quantification_files_location': "The transformed / aggregated / combined quantification values for each peptide, calculated from the quantification values from the previous section (5.1.1) (e.g. medians, averages, standard deviations, super-medians, fold changes, p-values etc…). Also provide the values representing their statistical estimation of confidence (see above). The values can be provided either by a URI or the location of the resulting data files, or their availability.",
        'raw_quantification_values': "The calculated quantification value for each protein and for each protein ambiguity group and also values representing their confidence (see above).",
        'raw_quantification_files_location': " URI or the location of the resulting data files.",
        'transformed_quantification_values': "The transformed / aggregated / combined quantification values for each protein, calculated from the quantification values above (e.g. medians, averages, standard deviations, super-medians, fold changes, p-values etc…), at group level. Also provide the values representing their statistical estimation of confidence (see above).",
        'transformed_quantification_files_location': "URI or the location of the resulting data files, or their availability.",
        /*MASS SPECTROMETRY INFORMATICS*/
        'msi_responsible_person': "The (stable) primary contact person for this data set; this could be the experimenter, lab head, line manager etc.. Where responsibility rests with an institutional role (e.g. one of a number of duty officers) rather than a person, give the official name of the role rather than any one person. In all cases give affiliation and stable contact information. ",
        'msi_software': "For each software used: The trade name of the software used for the identification and/or characterization work, together with the version name or number according to the vendor’s nomenclature and vendor name",
        'msi_customizations': "Any (i.e. affecting behavior) modification made to the original code and functionality of the software",
        'msi_software_availability': "The references of the vendor or public url if a publicly available version has been used.",
        'msi_files_description': "Description for the data generated.",
        'msi_files_location': "The location of the data generated. If made available in a public repository, describe the URI (for instance an url, or the url of the repository and the information on how to retrieve the data). If not made available for public access, describe the contact person reference or source and the internal coordinates of the data.",
        'msi_inputdata_description': "Provide a short description that can refer to the data in the experiment (e.g. LC-MS run1). Specify if the submitted data are raw full traces (either as proprietary binary format or exported readable format) or if they are peaklists files. In all cases, specify the original format (i.e. Bruker .yep, Applied Biosystems .wiff, mzData, mzML, dta (Sequest format), mgf (Mascot generic format) and mzXML (Institute for Systems Biology xml format), other…) ",
        'msi_database_queried': "The description and version of the sequence databank(s) queried, If the databank(s) is/are not available on the web, describe the content of this/these databank(s), including the number of sequences. ",
        'msi_taxonomical_restrictions': "Specify the amplitude of the subset of the databank(s) (for instance, “mammals”, a NCBI TaxId, a list of accession numbers). Specify the number of entries searched. ",
        'msi_tool_description': "Descriptor of the scoring algorithm in the search engine (such as ESI-TRAP in Mascot, ESI – ion trap HCTultra in Phenyx, ESI-ION-TRAP in MS-Tag, Ion Trap (4 Da) in X !Tandem, etc) ",
        'msi_cleavage_agents': "Describe the cleavage agent as available on the search engine. If the cleavage agent rules have been defined by the user,  describe the cleavage rules)",
        'msi_missed_cleavages': "Allowed maximum number of cleavage sited missed by the specified agent during the in-silico cleavage process.",
        'msi_cleavage_additional_params': "This includes, for instance the consideration of semi-specific cleavages (occurring on only one terminus), or other parameter that is relevant to the generation of peptides. ",
        'msi_permissible_aminoacids_modifications': "Specify the amino acid modifications that have to be considered in the search, according to the search engine list and mode of consideration (fixed or variable for instance). If the user has added custom modification to the predefined lists, specify the modification and the associated rules.",
        'msi_precursorion_tolerance': "For tandem MS queries, specify the mass tolerance applied to precursor ions and to fragment ions submitted to the search engine (when applicable).",
        'msi_pmf_mass_tolerance': "For PMF and other MS queries, specify the mass tolerance applied to the search engine (when applicable) ",
        'msi_thresholds': "Describe the parameters associated to the selection of peptide and protein hits retained in the output. These might include, for instance, a minimal score, a maximum p-value, a maximum number of proteins in the output, a minimum number of peptides to match a protein, etc. ",
        'msi_otherparams': "Any application-specific parameters that are to be set for a search and that have an impact on the interpretation of the result for that experiment.",
        'msi_accession_code': "Protein Accession Code such as a Uniprot-SwissProt AC number (P34521) (not an ID such as YM45_CAEEL), or a ncbi gi number (12803681). In case of the concept of protein group is applied (i.e. a list of protein that share a number of identical peptides among those identified), the description of the protein group might replace the list of individual Accession Codes ",
        'msi_protein_description': "Protein description field from the database.",
        'msi_protein_scores': "Values as reported by the search engin",
        'msi_validation_status': "For all protein hits in the search, specify if accepted without post-processing of search engine/de-novo interpretation (accept raw output of identification software) or if manually accepted as valid or as rejected (false positive). ",
        'msi_different_peptide_sequences': "Do consider the number of different peptide sequences. A peptide identified as unmodified and for instance containing an oxidized Methionine is counted as one. A peptide with one missed cleavage and one included peptide with no missed cleavages are considered as two",
        'msi_peptide_coverage': "Expressed as the number of amino acids spanned by the assigned peptides divided by the sequence length. ",
        'msi_pmf_matched_peaks': "Describe the number of matched m/z values associated to the identified protein, and the number of unmatched signals (or the total number of m/z values in the original spectrum/spectra)",
        'msi_other_additional_info': "This might include retention time, multiplicity of peptide sequence occurrences, flanking residues, etc",
        'msi_sequence': "Primary sequence of the matched peptides (include number of missed cleavages or if the peptide is issued from semispecific cleavage or fully unspecific cleavage).\00",
        'msi_peptide_scores': "Scores values and any associated statistical information as available in the output",
        'msi_chemical_modifications': "Occurrence and position of amino acid modifications, being artefactual (such as oxidized Met or Carbamidomethylated Cys), post-translational (such as myristoylated amino acid), issued from an amino acid mutation or a frame shift with respect to the sequence in the database. When a choice among possible isobaric modifications are reported, a justification should be applied (I/L, acetylation/trimethylation ",
        'msi_spectrum_locus': "Reference to the experimental spectrum. ",
        'msi_charge_assumed': "Description of the precursor ion charge state and mass deviation (either expressed as m/z difference or as recalculated mass difference). ",
        'msi_quantitation_approach': "Describe the experimental protocol used for the quantitation experiment (refer to the sample prep section) ",
        'msi_quantitation_measurement': "Specify the way the quantitation is measured (e.g. integration of signals, use of signal intensity) ",
        'msi_quantitation_normalisation': "Describe the method and software: What are the input intensity values, how have they been filtered, transformed, normalised and processed",
        'msi_quantitation_replicates_number': "As part of the protocol. ",
        'msi_quantitation_acceptance': "Describe the evaluation method applied to the quantitation software (or manual calculation) result. Describe the acceptance criteria and measures of variability. ",
        'msi_quantitation_error_analysis': "Include estimates of uncertainty and error tolerances. Include description of treatment of relevant systematic error effects and the treatment of random error issues (such as labelling efficiency, instrument detector saturation, presence of non-unique peptides in MS/MS spectra, interference of distinct peptides with overlapping isotopes or other known or suspected sources of quantification outliers)",
        'msi_quantitation_control_results': "In case control experiments have been made, or in case internal controls have been incorporated in the protocol, describe the result of the analysis of these controls.",
        'msi_interpretation_assessment': "Describe the approach the software (when applicable) used for assessing confidence to the identification and quantitation (when applicable).",
        'msi_interpretation_results': "Describe the result of any applied statistics to estimate false positive rate in case of “large scale experiments”",
        'msi_interpretation_inclusion': "Provide access to the output data"
    },
    treatmentcreationwindows: {
    },
    regionselectorwindow: {
        region_name: "<b>[required]</b> Identify the region. E.g. <i>GRCh37 Ensembl Human genome</i>, <i>UCSC Mus musculus genes GTF file</i>...",
        source: "<b>[required]</b> Identify the source of the region data. E.g. <i>Downloaded from TopHat webpage (Index and annotation downloads section), http://tophat.cbcb.umd.edu/igenomes.shtml</i>",
        files_location: "<b>[required]</b> Location of the region files."
    }
};

showHelpTips = function (view) {
    var name = "";

    if (view instanceof SL.view.AnalysisViews.RAWDataViews.ExtractionMethodView) {
        name = "extractionmethodview";
    } else if (view instanceof SL.view.AnalysisViews.IntermediateDataViews.IntermediateDataView || view instanceof SL.view.AnalysisViews.ProcessedDataViews.ProcessedDataView) {
        name = "stepview";
    } else if (view instanceof SL.view.AnalysisViews.IntermediateDataViews.SpecificDetailsPanel) {
        name = "intermediate_data_specificdetailspanel";
    } else if (view instanceof SL.view.AnalysisViews.ProcessedDataViews.SpecificDetailsPanel) {
        name = "processed_data_specificdetailspanel";
    } else {
        name = view.$className.lastIndexOf(".")
        name = view.$className.slice(name + 1).toLowerCase();
    }
    if (enableHelpTips === true && helpTips[name] === undefined) {
        return;
    }
    var setHelpTipsContent = function (item) {
        if (item.labelEl === undefined) {
            return;
        }
        if (item.name === undefined) {
            return;
        }
        var pos = item.labelEl.dom.innerHTML.indexOf('formfieldHelpTip');
        if(pos !== -1){
            return;
        }
        pos = item.labelEl.dom.innerHTML.indexOf(':');
        var labelContent = item.labelEl.dom.innerHTML.slice(0, pos + 1);

        if (enableHelpTips === true && helpTips[name][item.name] !== undefined) {
            labelContent += ' <i class="fa fa-question-circle formfieldHelpTip" data-qtip="' + helpTips[name][item.name] + ' "></i>';
        }

        item.labelEl.dom.innerHTML = labelContent;
    };
    var items = view.query('field');

    var fieldsContainer = view.query('fieldcontainer');
    for (var i = 0; i < fieldsContainer.length; i++) {
        items.push(fieldsContainer[i]);
    }
    Ext.Array.each(items, setHelpTipsContent);
}
